ckan/logic/action/get.py

```diff
@@ -2588,7 +2595,9 @@ def group_activity_list(context, data_dict):
     '''
     # FIXME: Filter out activities whose subject or object the user is not
     # authorized to read.
-    _check_access('group_show', context, data_dict)
+    data_dict['include_data'] = False
```

Comment:
this modifies the caller's data_dict, use `dict(data_dict, include_data=False)` in the call below instead

Reply:
good point. done.